README.md

```diff
@@ -1,23 +1,33 @@
 # ap_verify_ci_hits2015
 
-Data from HiTS (2015) for fast testing of alert production in the LSST DM stack.
+Data from HiTS (2015) for fast testing of alert production in the LSST Science Pipelines.
 
-Contains DECam data from the HiTS (2015) fields `Blind15A_40` and `Blind15A_42`.
+This and other `ap_verify` "datasets" are based on [ap_verify_dataset_template](https://github.com/lsst-dm/ap_verify_dataset_template).
```

Comment:
I'm not sure I understand why this is important. Is it just so that future maintainers know where to start?

Reply:
Yes, someone (Eric? Ian? I forget who) specifically requested this be mentioned because they were looking into making a new dataset and didn't realize this starting place already existed.